src/components/Card/Post/Expanded.js

```diff
@@ -122,7 +122,10 @@ export default asCard(
     const faviconSrc = `https://www.google.com/s2/favicons?domain=${encodedHref}`;
 
     return (
-      <ConditionalAnchor css={styles} className={classNames(className, 'post')}>
+      <ConditionalAnchor
+        css={styles}
+        className={classNames(className, 'post-expanded')}
```

Comment:
Is there a css class for `post-expanded`, I wasn't seeing it within the `GlobalStyles/index.js` file. Want to make sure that we aren't missing anything

Reply:
There isn't